scripts/apbs-bornprofile-extractDX.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import pandas as pd
+from gridData import Grid
+
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    '--input', help='input filename')
+parser.add_argument(
+    '--output', help='output filename. .dx is auto added to name')
+parser.add_argument(
+    'grid', type=float, help='grid interval')
+parser.add_argument(
+    'gridL', type=float, help='loose grid interval')
+
+args = parser.parse_args()
+
+filename = args.input
+outname = args.output
+
+# Grid interval
+grid_int= args.grid
+
+# Edge offset for Grid
+offset = grid_int/2.0
+
+# For losse edge
+grid_intL = args.gridL
+offsetL = grid_intL/2.0
+
+df = pd.read_csv(filename,delim_whitespace=True)
+df = df.to_numpy()

```

Comment:
Instead of using pandas, just use the numpy function; pandas isn't a dependency.

(However, if integrated with existing code, you'll already have the data.)

Reply:
Also, calling a nump array `df` is confusing.